__tests__/app.test.js

```diff
@@ -311,6 +312,321 @@ describe("app", () => {
+            },
+            {
+              author: "icellusedkars",
+              title: "A",
+              article_id: 6,
+              topic: "mitch",
+              created_at: "2020-10-18T01:00:00.000Z",
+              votes: 0,
+              comment_count: 1,
+            },
+            {
+              author: "icellusedkars",
+              title: "Sony Vaio; or, The Laptop",
+              article_id: 2,
+              topic: "mitch",
+              created_at: "2020-10-16T05:03:00.000Z",
+              votes: 0,
+              comment_count: 0,
+            },
+            {
+              author: "butter_bridge",
+              title: "Moustache",
+              article_id: 12,
+              topic: "mitch",
+              created_at: "2020-10-11T11:24:00.000Z",
+              votes: 0,
+              comment_count: 0,
+            },
+            {
+              author: "butter_bridge",
+              title: "Living in the shadow of a great man",
+              article_id: 1,
+              topic: "mitch",
+              created_at: "2020-07-09T20:11:00.000Z",
+              votes: 100,
+              comment_count: 11,
+            },
+            {
+              author: "butter_bridge",
+              title: "They're not exactly dogs, are they?",
+              article_id: 9,
+              topic: "mitch",
+              created_at: "2020-06-06T09:10:00.000Z",
+              votes: 0,
+              comment_count: 2,
+            },
+            {
+              author: "rogersop",
+              title: "Seven inspirational thought leaders from Manchester UK",
+              article_id: 10,
+              topic: "mitch",
+              created_at: "2020-05-14T04:15:00.000Z",
+              votes: 0,
+              comment_count: 0,
+            },
+            {
+              author: "rogersop",
+              title: "Student SUES Mitch!",
+              article_id: 4,
+              topic: "mitch",
+              created_at: "2020-05-06T01:14:00.000Z",
+              votes: 0,
+              comment_count: 0,
+            },
+            {
+              author: "icellusedkars",
+              title: "Does Mitch predate civilisation?",
+              article_id: 8,
+              topic: "mitch",
+              created_at: "2020-04-17T01:08:00.000Z",
+              votes: 0,
+              comment_count: 0,
+            },
+            {
+              author: "icellusedkars",
+              title: "Am I a cat?",
+              article_id: 11,
+              topic: "mitch",
+              created_at: "2020-01-15T22:21:00.000Z",
+              votes: 0,
+              comment_count: 0,
+            },
+            {
+              author: "icellusedkars",
+              title: "Z",
+              article_id: 7,
+              topic: "mitch",
+              created_at: "2020-01-07T14:08:00.000Z",
+              votes: 0,
+              comment_count: 0,
+            },
+          ]);
+        });
```

Comment:
What about filtering by a topic that exists but doesn't have any articles associated (200 oK empty arr)